Anne-Hyacinthe/exercises/answers.js

```diff
@@ -0,0 +1,47 @@
+// Question 1: Looping a Triange
```

Comment:
Great job on this section, Anne! Each of these solutions is well-written and DRY (As in, you Do Not Repeat Yourself). However, there are small details that I want you to begin paying attention to. Your indentation, spacing between operators... all of that matters and sends a message about your conscientiousness.